pkg/expr/nodes.go

```diff
@@ -500,8 +525,90 @@ func WideToMany(frame *data.Frame) ([]mathexp.Series, error) {
 		if err != nil {
 			return nil, err
 		}
+		if fixSeries != nil {
+			fixSeries(s, frame.Fields[tsSchema.ValueIndices[0]])
+		}
 		series = append(series, s)
 	}
 
 	return series, nil
 }
+
+type seriesFixFunction = func(series mathexp.Series, valueField *data.Field)
+
+func checkIfNeedsToBeFixed(frames []*data.Frame) seriesFixFunction {
+	// determine whether all potential series have 0 labels. If there is at least one value field with non-zero labels, the slice will be empty
```

Comment:
This is a little bit trickier in alerting: what if Graphite returns 1 series, and after several evaluations it returns 2 or more... this will cause alerting to get confused.

Reply:
We may need to limit it to Graphite plugin type for now then... or graphite + some specific list.